### PR TITLE
Contributor agreement and copyright update.

### PR DESCRIPTION
**Pull request by hjoliver**
One for you @dpmatthews!

### CODE REVIEW

**Comment by hjoliver** [comment]
Note the italicised sentence in CONTRIBUTING.md - I think that should be a sufficient measure for "backward compatibility", no?

**Comment by hjoliver** [comment]
(Discussing the exact wording offline.)

**Comment by matthewrmshin** [comment]
Failing the code style test!

**Comment by hjoliver** [comment]
Doh. Fixed.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+section of this file as part of their first pull request. The developers who
+review each pull request are responsible for checking that the contributor's
+name is listed in this file before merging the new branch into *master*.
+
+## Code Contributors
+
+The following people have contributed to this code under the terms of
+the Contributor Licence Agreement and Certificate of Origin detailed
+below (_except for the parenthesised names, which represent contributions
+from outside of NIWA and Met Office that predate the explicit introduction of
```
Comment (dpmatthews) [inline CONTRIBUTING.md]: Should be "the Met Office"

**Comment by hjoliver** [comment]
(Done - also in in-source Copyright file-headers)

**Comment by dpmatthews** [comment]
Better to leave it out of the copyright notices - e.g. we don't have it Rose.
Sorry.

**Comment by hjoliver** [comment]
You're hard to please today! Try it for size now.

**Comment by matthewrmshin** [comment]
Not sure I want to cherry pick this one. Does it need to go to 7.7.2?

**Comment by hjoliver** [comment]
Good point.  Change it to 7.8.0.  It's on master now, so new contributors will still need to agree to it anyway.

**Comment by hjoliver** [comment]
(changed milestone)

**Pull request merged**